pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -229,6 +229,12 @@ stdenv.mkDerivation rec {
     addOpenGLRunpath $out/cuda_sanitizer_api/compute-sanitizer/*
     addOpenGLRunpath $out/cuda_sanitizer_api/compute-sanitizer/x86/*
     addOpenGLRunpath $out/target-linux-x64/*
+  '' + ''
+    # Prune broken symlinks which can cause problems with consumers of this package.
```

Comment:
Can you please make this a Nix comment instead of a shell comment? Having these comments in the shell code means that they impact the derivation hash which means that it will cause rebuilds the next time we have to change the comment.